package.json

```diff
@@ -27,8 +27,8 @@
     "testURL": "http://localhost"
   },
   "peerDependencies": {
-    "react": "15.x || 16.x || 16.4.0-alpha.0911da3",
-    "react-dom": "15.x || 16.x || 16.4.0-alpha.0911da3"
+    "react": "15.x || 16.x || 16.4.0-alpha.0911da3 || ^17.0.1",
+    "react-dom": "15.x || 16.x || 16.4.0-alpha.0911da3 || ^17.0.1"
```

Comment:
@alexandernst. Yuck... unfortunately, I think you are just adding to this mess. A Git blame for this line shows that the previous author wrote a commit message: ["Yeah I dunno"](https://github.com/reach/router/commit/970544970f94e56d3f303f4d1dd6489bf60286a9#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519). My 2 cents is that the peer dependencies should be cleaned up, using just one wild-card format. `^x.x.x` is more commonly used and also explicit. This would be far cleaner:

```
"peerDependencies": {
  "react": "^15.0.0 || ^16.0.0 || ^17.0.0",
  "react-dom": "^15.0.0 || ^16.0.0 || ^17.0.0"
},
```

For reference, here are some similar repo's addressing this issue: [material-ui](https://github.com/mui-org/material-ui/pull/23311/files/e9a9cdb22db37b6b08abaec8cdb7c01e6638d7a2). 

Alternatively, [react-router]( https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/package.json#L39-L41) doesn't even need to change any code to support React 17; however, I agree with Martin Fowler and others that ["To Be Explicit"](https://martinfowler.com/tags/clean%20code.html) is better.

Reply:
Another way to do it is `"15 - 17"`